bookie/tests/test_auth/test_api_base.py

```diff
@@ -123,3 +123,28 @@ def test_login_null(self):
         self.assertTrue(
             'Failed login' in str(res),
             "Should have 'Failed login' in the resp: " + str(res))
+
+    def test_redirect_for_authenticated_user(self):
```

Comment:
`test_login_url` test on Line26 already does that. That's why, I didn't add another test case.